region/norfair/east.json

```diff
@@ -5633,6 +5635,7 @@
                       "shinesparkFrames": 33
                     }},
                     {"heatFrames": 500},
+                    {"lavaFrames": 300},
```

Comment:
I think its just under 3s.
And the heat frame count is 7s.

Reply:
This is including the time to get to the surface, while in the shinespark, right?

Reply:
Yeah, I timed it by updating the room timer when I entered the lava and when I exited the lava.